CMakeLists.txt

```diff
@@ -5,12 +5,16 @@ set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake-scripts)
 
 option(BUILD_EXAMPLES "Build example binaries" ON)
 option(BUILD_ALLEGRO4 "Build Allegro4 support" ON)
+option(USE_SSE "Use SSE instructions" ON)
```

Comment:
Ideally yes, but I admit I wasn't sure how to do it, so I went for the simple solution as a workaround.

I've googled a bit for this and most answers involve adding long CMake modules that are not always cross-platform and try to determine all the supported levels of SSE instructions.

But a hardcoded architecture check might be good enough already since it's just about SSE and not about whether we can use SSE3 or SSE4.1.